cmd/insetmap/insetmap/main.go

```diff
@@ -17,7 +17,8 @@ var (
 	entryFlag  = flag.String("entry", "", "which entry to use from the config file. If not set, will be the default entry, or the first entry.")
 	mdgid      = flag.String("mdgid", "V795G25492", "mdgid to generate svg for.")
 	configFile = flag.String("config", "config.toml", "config file")
-	InsetMaps  map[string]*insetmap.Inset
+	boundary   = flag.Bool("boundary", false, "generate a boundarry map instead of an inset map")
```

Comment:
This description seems a bit odd to me as the "boundary" map is an inset map.

Reply:
boundary you thinking it should say instead of an adjoining sheets map?